samples/language/CopyAndUpdateOperator.qs

```diff
@@ -19,15 +19,18 @@ namespace MyQuantumApp {
         // `new_array` is an array with values `[10, 11, 100, 13]`.
         // `array` is unchanged.
         let new_array = array w/ 2 <- 100;
+        Message($"Updated array: {new_array}");
 
         // `new_array` is an array with values `[10, 100, 12, 200]`.
         // `array` is unchanged.
         let new_array = array
             w/ 1 <- 100
             w/ 3 <- 200;
+        Message($"Updated array: {new_array}");
 
         // `new_struct` is a Pair with value `Pair(20, 100)`.
         // `struct` is unchanged.
         let new_struct = struct w/ second <- 100;
+        Message($"Updated struct: {new_struct}");
```

Comment:
```suggestion
        Message($"Updated struct: (first:{new_struct::first}, second:{new_struct::second})");
```
Hmm, does this work for you. I get compile time error when printing UDT. Seems to behaviour change from classic QDK :-(. I encountered the issue during Superdense Coding Kata migration
**Not working**
![image](https://github.com/microsoft/qsharp/assets/40084144/5e19a384-1ce4-487e-b144-93e950743993)

**Working**
![image](https://github.com/microsoft/qsharp/assets/40084144/d0a6f7ea-3a0d-4053-bdc3-72ee4af54fd1)

Reply:
> PS: Not working on current playground. [Playground link](https://microsoft.github.io/qsharp/?code=H4sIAAAAAAAACq2TMW%2FbMBCFd%2F%2BKh7iDBSimpGxyDLRoOnQwmqDxFBQVK51tojIpkFQEIfB%2FL0TKku02Gdpyko73Hu87HhljmOIr31clTRhj%2BKiqFlwWWFcFt4QvFWlule42XcIUd2RyLSorlHSRxx0hP8pqL1O9DELiYQphUBsqYBX2%2FCeBe4HagKPgljsfY3Wd21pTiFJ0SRJca95CaazvHsNTfw4j5LYkUEl7krY7x%2B48Qq4qQQWeSRuhZHeI3dHoPp9IvidT8Zywah9qLm29%2F1BVeJkAgKTGthXhnguNJWYboY1Fis%2FShjCUK1n4v2Dh8t9%2Fkla390pIOwtcxLN3R6%2B4kLMAKdZS2N6%2FWyXZHm2JpzgKEcch4iREfPNtcZblq8bSlTNLohBJHIwpjCFrWIaNKkvVUIEfrYPNbq8z3wdz0jXzW9vmZ06Smu%2BurKy7sKH9jbA7PPOyJoNsLDeKXL3Zuceor2W%2B43JLxfyMaDgEy6M%2FQ4Lb685xJFuRMXxLs3dXfhALn5ziZTA4XAWLyV8CuOKTEEkU%2FTvBsNuthiHuaS7jN108%2BS%2BUfi48ph%2FVERLZMCxxFAUXeCfKN%2FiGues%2FGnac%2FcuLYuwPFF7VY%2Fgfx%2FE691Hh31t6IkxTFzocX9%2F5no8dgqFLh8nhFwUijbHVBAAA&profile=unrestricted)